### ISSUE TITLE
Unused import is not reported

### ISSUE THREAD

**Issue by Ilya-Gh** [pinterest/ktlint#405]
Unused import a of static function is not reported if the function is used referenced with its parent class.

Android Studio/IDea does report the import as unnecessary using [KotlinUnusedImportInspection](https://github.com/JetBrains/kotlin/blob/master/idea/src/org/jetbrains/kotlin/idea/inspections/KotlinUnusedImportInspection.kt):

![image](https://user-images.githubusercontent.com/10222326/56818066-8edc0600-684f-11e9-8f6f-d042be96d2f3.png)

The [following test added to `NoUnusedImportsRuleTest`](https://github.com/Ilya-Gh/ktlint/commit/eeb77aef20e82f57ca309d66d674f06efaed0e8d#diff-2b4cd66b8a3a6ecff83990f820622d1dR196) currently [fails](https://travis-ci.org/Ilya-Gh/ktlint/builds/524993641):

```kotlin
@Test
fun testParentPackImport() {
    assertThat(
        NoUnusedImportsRule().lint(
            """
            import A.b

            fun main() {
                A.b
            }
            """.trimIndent()
        )
    ).isEqualTo(
        listOf(
            LintError(1, 1, "no-unused-imports", "Unused import")
        )
    )
}
```
The error:
```
java.lang.AssertionError: 
Expecting:
 <[]>
to be equal to:
 <[LintError(line=1, col=1, ruleId=no-unused-imports, detail=Unused import)]>
but was not.
	at com.pinterest.ktlint.ruleset.standard.NoUnusedImportsRuleTest.testParentPackImport(NoUnusedImportsRuleTest.kt:208)
```

**Comment by shashachu** [comment]
Re-opening because I reverted the fix, as it caused a couple regressions with false-positives.
